home/ivp-planning/settings/global-settings.md

```diff
@@ -1,26 +1,21 @@
 # Global Settings
 
-The global settings offer different options to adjust the [user interface](../user-interface/) and the [floor plan](../user-interface/the-floor-plan.md). They can be tweaked by using the [settings panel](../user-interface/settings-panel.md).
-
 ![](../../../.gitbook/assets/iVP\_settings\_global\_settings.jpg)
 
 ## The options explained:
 
-* Global Unit:&#x20;
-* Rotation Amount:
-* Snapping Distance:
-* Path Snapping Angle:
-* Ground Plan X:
-* Ground Plan Y:
-* Wall Height:
-* Camera Speed:
-* UI Scale:\
-
-* Grid:
-* Coordinate System:
-* Animations:
-* PDF Floorplan:
-* Overhead View:
-* Walls:
-* Ceiling:
-* Project Info:
+* Unit: Determines the unit of measurement
+* Floor Material: Changes the floor material (e.g. invisible, concrete)
+* Rotation Amount: Determines the rotation amount of the Rotate Clockwise & Rotate Counterclockwise options
+* Snapping Distance: Sets the snapping precision for grid-aligned objects
+* Ground Plan X: Adjusts ground plan width
+* Ground Plan Y: Adjusts ground plan height
+* Wall Height: Adjusts wall heights
+* Camera Speed: Adjusts the movement speed of the camera in the 3D panel
+* UI Scale: Adjusts UI scale
+* Grid: Toggles the grid in the 2D panel
+* Grid Origin: Toggles the origin marker
+* Animations: Allows animations to be visible in the 3D panel
+* Overhead View: Allows animations to be visible in the 2D panel
```

Comment:
This also makes the 2D panel render in more detail. We should also note that this has a negative impact on performance.